Components/IconTypes/IconType_customvalue/customvalue.lua

```diff
@@ -0,0 +1,130 @@
+﻿local TMW = TMW
+if not TMW then return end
+
+local L = TMW.L
+
+local print = TMW.print
+
+
+local Type = TMW.Classes.IconType:New("customvalue")
```

Comment:
Suggestion for a better type identifier: `luavalue`. More concise and clearer what it is doing. Make sure to update file names accordingly.